index.html

```diff
@@ -0,0 +1,286 @@
+                  <div class="skills__container">
+                    <p>HTML</p>
+                  </div>
+                  <div class="skills__container">
+                    <p>CSS</p>
+                  </div>
+                  <div class="skills__container">
+                    <p>JavaScript</p><br/>
+                  </div>
+              </div>
+              <div class="Portfolio__project__layout">
+                <div class="Portfolio__project__design">
+                  <div class="Portfolio__project__content">See Project </div>
+                </div>
+              </div>
+            </div>
+            </div>
+          </section>
+          <section>
+            <div class="navigation__work">
+              <img src="img/work/port4.png" class="portfolio_f_image" alt="" />
+              <div class="portfolio__img__content">
+              <h3 class="portfolio__tonic">Multi-Post Stories</h3>
+              <div class="feature__container">
+                <div class="feature__container_side">
+                  <p class="category__portfolio">CANOPY</p>  
+                </div>
+                  <div class="canopy__design"></div>
+                  <div class="canopy__layout">
+                   <p class="canopy__backend">Back End Dev</p>
+                   
+                  </div>
+                  <div class="backend__design">
+                  <p class="category__port">2015</p>
+                  </div>
+              </div>
+              <p class="navigation__paragraph">
+                A daily selection of privately personalized reads; no accounts or
+                sign-ups required</p>
+              <div>
+                <div class="skills">
+                  <div class="skills__container">
+                    <p>HTML</p>
+                  </div>
+                  <div class="skills__container">
+                    <p>CSS</p>
+                  </div>
+                  <div class="skills__container">
+                    <p>JavaScript</p><br/>
+                  </div>
+              </div>
+              <div class="Portfolio__project__layout">
+                <div class="Portfolio__project__design">
+                  <div class="Portfolio__project__content">See Project </div>
+                </div>
+              </div>
+            </div>
+            </div>
+          </section>
+          <section>  
+          <div class="Portfolio__About__MySelf">
+            <h4 class="About__MySelf__header">About Myself</h4>
+            <div class="About__MySelf__Content">
+              <p>Hello I’m a software developer! I can help you build a product , feature or website. Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hesitate to contact me.</p>
+            </div>
+            <div class="About__MySelf__layout">
+              <div class="About__MySelf__design">
+                <div class="About__MySelf__Connect">LET'S CONNECT</div>
```

Comment:
- [ ] Kindly use a `hx` tag for this heading as this is good for page structure, e.g `h2` `h3` and `h4`